docs(how-to): rpn-san (#219)
scaleway/docs-content: dedibox-network/rpn/how-to/find-rpn-gateway.mdx

---
title: How to find your RPN gateway
description: This page explains how to find your RPN gateway
dates:
  validation: 2021-11-10
  posted: 2021-11-10
---

Your RPNv1 gateway may differ from one Dedibox to another for techical reasons.
It is normally pushed by DHCP, and and configured automatically.

> But in some cases you may need to know it.

Reviewer: ```suggestion
However, in some cases you may need to know the details of the gateway.
```

Reviewer: I'd put all these sentences in one paragraph.

<Message type="requirement">

* You have an account and are logged into the [Dedibox Console](https://console.online.net)
* You have a Dedibox dedicated server with RPN interface

</Message>

You can find the RPNv1 gateway for your Dedibox in [rescue mode](/dedibox/dedicated-servers/how-to/use-rescue-mode/) by typing:

```
# ip route show
10.90.0.0/XX via X.X.X.X ...
```

If you do not know your RPN netmask, you can find it by using IPCALC. The gateway is always the first IP of the subnet.
```
# ipcalc -b [SET_HERE_YOUR_RPN_IP]/[NETMASK CIDR] | grep HostMin
HostMin: X.X.X.X
```
The gateway is in place of `X.X.X.X`.

<Message type="tip">

The gateway does not ping, this is normal behavior.

> </Message>

Reviewer: It's a bit unclear for me what is part of each command, what is the output and what bits need to be replaced with users own values. Maybe something like:

1. Enter rescue mode (link to how)

2. Enter the following command to display your RPNv1 gateway (detailing any substitutions to make):

```
command
```

Your output should be similar to the following and shows you bla bla

```
output
```

3. Enter the following command to display your RPN netmask (detailing any substitutions to make):

```
ipcalc bla bla bla
```

Your output should be similar to the following and shows you bla bla

```
output
```

I don't know if the "gateway does not ping" message is a consequence of the ipcalc command or the ip route show command?

Do they need the netmask in order to carry out the first command for the RPNv1 gateway?

Author: I will update this. After proofreading it does not make much sensse

---
meta: 
  title: How to find your RPNv1 gateway
  description: This page explains how to find your RPNv1 gateway
content:
  h1: How to find your RPNv1 gateway
  paragraph: This page explains how to find your RPNv1 gateway
dates:
  validation: 2021-11-10
  posted: 2021-11-10
---

Your RPNv1 gateway may differ from one Dedibox to another for techical reasons.
It is normally pushed by DHCP, and and configured automatically.
However, in some cases you may need to know the details of the gateway.

You can find the RPNv1 gateway for your Dedibox in [rescue mode](/dedibox/dedicated-servers/how-to/use-rescue-mode/).

<Message type="requirement">

* You have an account and are logged into the [Dedibox Console](https://console.online.net)
* You have a Dedibox dedicated server with RPN interface

</Message>

1. Reboot your Dedibox into [rescue mode](/dedibox/dedicated-servers/how-to/use-rescue-mode/). 

  <Message type="important">

>     Your applications hosted on the Dedibox become unavailable as long a your server is booted in rescue mode. 

Reviewer: ```suggestion
    Applications hosted on the Dedibox become unavailable while your server is in rescue mode. 
```

  </Message>

> 2. Run the `ip route show` command to display your RPNv1 gateway. An output as the following displays:

Reviewer: ```suggestion
2. Run the `ip route show` command to display your RPNv1 gateway. An output like the following displays:
```
